src/librustc_trans/trans/consts.rs

```diff
@@ -1032,7 +1037,7 @@ pub fn trans_static(ccx: &CrateContext,
             expr,
             empty_substs,
             None,
-            TrueConst::Yes,
+            TrueConst::No,
```

Comment:
I had to change this because otherwise, statics like this were causing compiler errors:
```
const A: [i32; 1] = [0];
static B: i32 = A[0];
```